Belot/Belot.Server/SignalR/BelotHub.cs

```diff
@@ -169,7 +177,10 @@ public async Task Announce(GameAnnouncementRequest request)
 
```

Comment:
judgeManager.GetJudge(request.GameId) repeats. Assign it to local variable where appropriate

Reply:
Yes, will refactor this.